app/assets/javascripts/visualizations/highvis/photos.coffee

```diff
@@ -71,20 +80,26 @@ $ ->
               dset = data.metadata[ds]
               do(tmp, dset) ->
                 figure = """<div class='p_item'>
-                  <img id='pic_#{i}' src="#{tmp.tn_src}" class='caroucell'/> <br>
-                  <div class="caption_wrap"><span class="caption">Data Set: #{dset.name}(#{dset.dataset_id})
-                  </span></div> <br>
+                  <img id='pic_#{i}' src="#{tmp.tn_src}"/> <br>
+                  <div class="caption_wrap">
+                    <span class="caption">
+                      Data Set: #{dset.name}(#{dset.dataset_id})
```

Comment:
Nit picky, but could you add a space between the data set name and the id?  That's how it looks in the vis controls :)